services/server/src/handlers/contactRequest.ts

```diff
@@ -14,9 +21,20 @@ const createPayload = zod.object({
   message: string,
 });
 
-async function create(req: Request, res: Response, _: NextFunction) {
+async function create(req: Request, res: Response, next: NextFunction) {
   const payload: IContactRequest.CreatePayload = createPayload.parse(req.body);
 
+  const validations = [
+    isValidName(payload.name),
+    isValidEmail(payload.email),
+    isValidContactRequestTitle(payload.title),
+    isValidContactRequestMessage(payload.message),
+  ];
+
+  const invalidInput = validations.find((v) => v !== true);
+  if (invalidInput !== undefined)
+    return next(apierror(invalidInput as ApiErrorCode, 400));
```

Comment:
I konw that your code is safe but to avoid casting we can do it like this:

```suggestion
for (const result of validations) {
    if (result !== true) return next(apierror(result, 400))
}
```